### PR TITLE
feat: update helm charts to support loki HA deployment

### PR DESCRIPTION
**Pull request by jleach**
Updates the helm chart so that Loki can be deployed in a high-availability configuration.

### CODE REVIEW

**Comment by jleach** [comment]
Currently testing in the "test" environment.

**Comment by jleach** [comment]
This is done and deployed. Just waiting to see how it holds up in our "test" environment.

File: devops/README.md [hunk 1]
```diff
@@ -20,34 +20,47 @@ Read more about Loki [here](https://grafana.com/oss/loki/).
...
+| proxyPassword    | The password for the Loki Proxy.This is part of the authentication credentials.                     |
+| minio_access_key | The access key associated with Minio                                                                |
+| minio_secret_key | The secret key associated with Minio                                                                |
 
-Once deployed there will be two pods running that can be verified with the following command:
+Once deployed there will be several pods running, depending on your replication count, that can be verified with the following command:
 
 ```console
-➜  vc-wallet-mobile ✗ oc get pods -l "app.kubernetes.io/name=logstack"
+➜  vc-wallet-mobile git:(feat/loki-ha) ✗ oc get pods -l "app.kubernetes.io/name=logstack"
```
Comment (bryce-mcmath) [inline devops/README.md:53]: this is v nitpicky but there are a few instances of vc-wallet-mobile rather than bc-wallet-mobile

**Review by bryce-mcmath — APPROVED** [review]
One nitpicky thing that doesn't need to be changed, everything else looks good.

**Pull request merged**